doc/development/developing-sql-dialect/developing_a_dialect.md

```diff
@@ -23,30 +23,35 @@ The name SQL dialect adapter is derived from the non-standard implementation par
-    |   core     |----------------------------|
-    |            |//// Virtual Schema API ////|
-    |------------|----------------------------|
-    |            |       JDBC  Adapter        |   Common JDBC functions
-    |  In this   |----------------------------|
-    | repository |///// SQL Dialect API //////|
-    |            |----------------------------|
-    |            |    SQL Dialect Adapter     |   Even out specifics of the source database
-    |------------|----------------------------|
-    |            |///////// JDBC API /////////|
-    |            |----------------------------|
-    |            |  PostgresSQL JDBC Driver   |   JDBC compliant access to payload and metadata
-    |  External  |----------------------------|
-    |            |// PostgresSQL Native API //|
-    |            |----------------------------|
-    |            |         PostgreSQL         |   External data source
-    '-----------------------------------------'
+The layers in the middle &mdash; i.e. everything that deals with translating between the source and Exasol &mdash; are provided in this repository.
+
+    .------------------------------------------------------.
+    |  Exasol                 |          Exasol            |
+    |   core                  |----------------------------|
+    |                         |//// Virtual Schema API ////|
+    |-------------------------|----------------------------|
+    | In vs-common-java       |    Base for all adapters   |   Foundation of an adapter development
+    |   repository            |----------------------------|
+    | In vs-common-jdbc       |       JDBC  Adapter        |   Common JDBC functions
+    |   repository            |----------------------------|
+    |                         |///// SQL Dialect API //////|
+    |                         |----------------------------|
+    | In dialect repositories |    SQL Dialect Adapter     |   Even out specifics of the source database
+    |-------------------------|----------------------------|
+    |                         |///////// JDBC API /////////|
+    |                         |----------------------------|
+    |                         |  PostgresSQL JDBC Driver   |   JDBC compliant access to payload and metadata
+    |  External               |----------------------------|
+    |                         |// PostgresSQL Native API //|
+    |                         |----------------------------|
+    |                         |         PostgreSQL         |   External data source
+    '------------------------------------------------------'
 
 For more information about the structure of the Virtual Schemas check the UML diagrams provided in the directory [model/diagrams](../../../model/diagrams). You either need [PlantUML](http://plantuml.com/) to render them or an editor that has PlamtUML preview built in.
 
+* [Virtual Schema Common Java Repository](https://github.com/exasol/virtual-schema-common-java)
```

Comment:
I suggest to use the same name as above then, either `vs-common-java-repository` or `Virtual Schema Common Java Repository`

Reply:
Good Idea. Unified them